py/torch_tensorrt/dynamo/conversion/_TRTInterpreter.py

```diff
@@ -629,35 +657,68 @@ def run(
         assert serialized_engine
 
         _LOGGER.info(
-            f"Build TRT engine elapsed time: {datetime.now() - build_engine_start_time}"
+            f"Build weight-stripped TRT engine elapsed time: {datetime.now() - build_engine_start_time}"
```

Comment:
Probably just leave this as is, if the user requests immutable_weights, it wouldnt apply